src/core/js/models/questionModel.js

```diff
@@ -240,11 +240,15 @@ define([
                 feedbackMessage: body
             });
         },
-        
+
         getFeedbackTitle: function() {
             return this.get('_feedback').title || this.get('displayTitle') ||  this.get('title') || "";
         },
 
+        isInteractive: function() {
```

Comment:
I think this would benefit from a comment along the lines of 'used to determine whether the learner is allowed to interact with the question component or not'.